src/helpers/misc.ts

```diff
@@ -389,7 +389,7 @@ export function getAddHeaderStartIndex(position: "before" | "after", base: numbe
 /**
  * Compares two objects.
  */
-export function deepEquals(o1: any, o2: any): boolean {
+export function deepEquals(o1: any, o2: any, ignoreFunctions?: "ignoreFunctions"): boolean {
```

Comment:
I must admit I don't like the fix.
Ignoring function is wrong and error prone. It might be correct in this very specific use-case, but as a general helper, it can't guarantee two objects are equal if we ignore functions, and this hack looks like something to workaround another design problem